xtask/codegen/src/generate_configuration.rs

```diff
@@ -258,7 +258,7 @@ pub(crate) fn generate_rules_configuration(mode: Mode) -> Result<()> {
             }
 
             /// Set the severity of the rule to its default.
-            pub fn set_default_severity(&mut self, group: RuleGroup, rule_name: &str) {
+            pub fn set_default_severity_if_off(&mut self, group: RuleGroup, rule_name: &str) {
```

Comment:
We should update the documentation of this function, its name is different, so maybe its semantics changed?